charmcraft/templates/init/charmcraft.yaml.j2

```diff
@@ -0,0 +1,9 @@
+# Learn more about config at: https://juju.is/docs/sdk/config
```

Comment:
This link should probably be: https://juju.is/docs/sdk/charmcraft-config

Reply:
Yes indeed, thanks! Done!